.github/workflows/codspeed.yml

```
@@ -16,17 +16,18 @@ jobs:
     steps:
       - uses: actions/checkout@v4
 
-      - uses: pdm-project/setup-pdm@v4
+      - name: Setup uv
+        uses: astral-sh/setup-uv@v3
         with:
-          python-version: '3.12'
-          cache: true
+          enable-cache: true
+
+      - name: Set up Python 3.12
+        run: uv python install 3.12
```

```suggestion

      - uses: actions/setup-python@v3
        with:
          python-version: "3.12"
```
This change is working with codspeed. May I push the changes?

---

Sure thing